src/helpers/accessiblity.ts

```diff
@@ -226,3 +226,14 @@ export function isElementExpanded(
   const { accessibilityState, 'aria-expanded': ariaExpanded } = element.props;
   return ariaExpanded ?? accessibilityState?.expanded ?? false;
 }
+
+export function isElementCollapsed(
+  element: ReactTestInstance
+): NonNullable<AccessibilityState['expanded']> {
+  const { accessibilityState, 'aria-expanded': ariaExpanded } = element.props;
+  return ariaExpanded !== undefined
```

Comment:
Why not `return (ariaExpanded ?? accessibilityState?.expanded) === false`

Reply:
It's the same, I just wanted to make sure the syntax will be correct for the default case. Changing it now